Mini-LISP/mini-lisp.h

```diff
@@ -77,5 +78,8 @@ extern S list(S, S);
 extern S list(S, S, S );
 extern S list(S, S, S, S);
 
```

Comment:
There was linkage errors because those errors functions.
(Moved from "test.h" to here)